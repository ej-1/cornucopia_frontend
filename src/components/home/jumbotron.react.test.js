import React from "react";
import Jumbotron from "../home/jumbotron";
import renderer from "react-test-renderer";

test("InfoBox gets rendered", () => {
  let jumbotronProps = {
    header: "Welcome to CornucopiaTrader!",
    text:
      "A trading service for simulation trading strategies based on \n \
            technical indicators using historical trade data.",
    buttonLink: "/",
    buttonStyle: "primary",
    buttonText: "Learn more"
  };

  const component = renderer.create(<Jumbotron {...jumbotronProps} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
