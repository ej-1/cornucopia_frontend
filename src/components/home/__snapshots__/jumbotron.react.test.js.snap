// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoBox gets rendered 1`] = `
<div
  className="col-md-12 col-xs-12"
>
  <div
    className="jumbotron"
  >
    <h1>
      <span>
        Welcome to CornucopiaTrader!
        <br />
      </span>
    </h1>
    <p>
      <span>
        A trading service for simulation trading strategies based on 
        <br />
      </span>
      <span>
                     technical indicators using historical trade data.
        <br />
      </span>
    </p>
    <p>
      <button
        className="btn btn-primary"
        disabled={false}
        type="button"
      >
        Learn more
      </button>
    </p>
  </div>
  ;
</div>
`;
